modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -6,129 +6,136 @@
+  prior.variances["seedling_mortality", 1] <- 1
+  taupriors <- list(tauA = 0.01, tauB = apply(prior.variances, 1, function(x) min(0.01, x)))
   
   ### Run the meta-analysis
-  trait.mcmc  <- pecan.ma(jagged.data, prior.distns, taupriors, j.iter = iterations, 
-                          outdir = pft$outdir, random = random)
+  trait.mcmc <- pecan.ma(jagged.data,
+                         prior.distns,
+                         taupriors, 
+                         j.iter = iterations, 
+                         outdir = pft$outdir, 
+                         random = random)
+  
   ### Check that meta-analysis posteriors are consistent with priors
-  for(trait in names(trait.mcmc)){
-    post.median    <- median(as.matrix(trait.mcmc[[trait]][,'beta.o']))
-    prior          <- prior.distns[trait, ]
-    p.ma.post      <- p.point.in.prior(point = post.median, prior = prior)
+  for (trait in names(trait.mcmc)) {
+    post.median <- median(as.matrix(trait.mcmc[[trait]][, "beta.o"]))
+    prior       <- prior.distns[trait, ]
+    p.ma.post   <- p.point.in.prior(point = post.median, prior = prior)
+    
     ## if inside 95%CI, ok.
-    if(p.ma.post <= 0.9995 & p.ma.post >= 0.0005){
+    if (p.ma.post <= 0.9995 & p.ma.post >= 5e-04) {
       if (p.ma.post <= 0.975 & p.ma.post >= 0.025) {
         logger.info("OK! ", trait, " posterior and prior are consistent:")
       } else {
         logger.warn("CHECK THIS: ", trait, " posterior and prior are inconsistent:")
       }
-    } else if (p.ma.post > 0.9995 | p.ma.post < 0.0005) {
-      logger.severe("NOT OK! ", trait," posterior and prior are probably not the same:")
+    } else if (p.ma.post > 0.9995 | p.ma.post < 5e-04) {
+      logger.severe("NOT OK! ", trait, " posterior and prior are probably not the same:")
       return(NA)
     }
     logger.info(trait, "P[X<x] =", p.ma.post)
   }
-   
+  
   ### Generate summaries and diagnostics
   pecan.ma.summary(trait.mcmc, pft$name, pft$outdir, threshold)
   
   ### Save the meta.analysis output
-  save(trait.mcmc, file = file.path(pft$outdir, 'trait.mcmc.Rdata'))
+  save(trait.mcmc, file = file.path(pft$outdir, "trait.mcmc.Rdata"))
   
   post.distns <- approx.posterior(trait.mcmc, prior.distns, jagged.data, pft$outdir)
-  save(post.distns, file = file.path(pft$outdir, 'post.distns.MA.Rdata'))
+  save(post.distns, file = file.path(pft$outdir, "post.distns.MA.Rdata"))
   
-  # Symlink to post.distns.Rdata (no "MA" identifier)
-  if(file.exists(file.path(pft$outdir, 'post.distns.Rdata'))) {
-    file.remove(file.path(pft$outdir, 'post.distns.Rdata'))
+  # Symlink to post.distns.Rdata (no 'MA' identifier)
+  if (file.exists(file.path(pft$outdir, "post.distns.Rdata"))) {
+    file.remove(file.path(pft$outdir, "post.distns.Rdata"))
   }
-  file.symlink(file.path(pft$outdir, 'post.distns.MA.Rdata'), file.path(pft$outdir, 'post.distns.Rdata'))
-
+  file.symlink(file.path(pft$outdir, "post.distns.MA.Rdata"), file.path(pft$outdir, "post.distns.Rdata"))
+  
```

Comment:
The two paths in here are used multiple times. It would be safer to define them as variables and use those here.